--- bin/catalina.sh	2020-06-09 13:29:34.154909672 +0200
+++ bin/catalina.sh	2020-06-09 13:30:59.283752883 +0200
@@ -114,6 +114,7 @@
 #                   signals. Default is "false" unless running on HP-UX in which
 #                   case the default is "true"
 # -----------------------------------------------------------------------------
+JAVA_OPTS="-Djava.awt.headless=true -Dfile.encoding=UTF-8 -server -Xms1536m -Xmx1536m -XX:NewSize=256m -XX:MaxNewSize=256m -XX:+DisableExplicitGC"
 
 # OS specific support.  $var _must_ be set to either true or false.
 cygwin=false
